.github/workflows/build_and_test.yml

```diff
@@ -1028,8 +1028,8 @@ jobs:
     needs: precondition
     if: fromJson(needs.precondition.outputs.required).tpcds-1g == 'true'
     name: Run TPC-DS queries with SF=1
-    # Pin to 'Ubuntu 20.04' due to 'databricks/tpcds-kit' compilation
-    runs-on: ubuntu-20.04
+    # Pin to 'Ubuntu 24.04' due to 'databricks/tpcds-kit' compilation
+    runs-on: ubuntu-24.04
```

Comment:
can we just use `ubuntu-latest` as others

Reply:
Updated it.